phpstan.neon

```
@@ -0,0 +1,5 @@
+includes:
+	- vendor/jangregor/phpstan-prophecy/src/extension.neon
+parameters:
+    ignoreErrors:
+        - '#^Cannot cast object|string to string$#'
```

I think we shouldn't ignore this error. Where this error occurs?

---

src/Logger.php:96

---

Could be a good idea to do the following?

```php
if (\is_object($message) && \method_exists($object, '__toString')) {
    $message = $message->__toString();
}
```